content/kubermatic/master/tutorials_howtos/aws_assume_role/_index.en.md

```diff
@@ -0,0 +1,47 @@
++++
+title = "How To run Clusters on AWS using assumed roles"
```

Comment:
I think the title should be changed, it doesn't really match other titles in the tutorials section and it's kind of inconsistent with capital letters. Why not call it "Deploy with AWS AssumeRole"

Reply:
There was a guide with that prefix, but it has been renamed since 2.18. I'll have another look :slightly_smiling_face: